server/index.js

```diff
@@ -1,9 +1,18 @@
 const express = require('express');
 const port = 4004;
-
 const app = express();
+const db = require('../database/database.js');
+const seeder = require('../database/seeding.js');
+const bodyParser = require('body-parser');
 
 app.use(express.static(__dirname + '/../client/dist'));
+app.use(bodyParser.urlencoded({extended: false}));
+app.use(bodyParser.json());
+
+app.get('/description/1000', (req, res) => {
+  var data = db.findOne({productId: 1000});
+  res.send(data);
```

Comment:
Optimally, you also want to return a status code along with the server response. So you can do something like `res.status(200).send(data);`. And you want to have a response in case the db query returns no results or fails.